packages/pilot/src/pages/LoggedArea/index.js

```diff
@@ -62,7 +62,13 @@ const LoggedArea = ({
           component={component}
         />
       ))}
-      <Redirect to={recipientId ? `/balance/${recipientId}` : '/balance/'} />
+      <Redirect
+        to={
+          recipientId ?
```

Comment:
alterar esse ternario para manter o padrão
```suggestion
          recipientId
          ? `/balance/${recipientId}`
          :           '/balance/'
```